### DISCUSSION TITLE
Opcode Information API

### DISCUSSION THREAD

**Discussion by cyrillicw** [wasmerio/wasmer#2621] [Ideas]
Is there a possibility to be notified about what opcode and with which parameters is executed? For example, if the sum of two numbers is computed, I would like to receive the next information: ```sum with a=17 and b=23```.

**Comment by Hywan** [comment]
Hello @cyrillicw,

Maybe this issue might be interesting for you https://github.com/wasmerio/wasmer/issues/1210. If yes, then please I let you close this issue as a duplicate. Otherwise, please can you elaborate about your needs :-)?

**Comment by cyrillicw** [comment]
It's almost what I am asking about. But as I understand it creates a log file with full information about the executed opcode but I would like to process such rows in some function without creating the file
